src/AbstractPolytope.jl

```diff
@@ -162,15 +162,19 @@ else
                       intersect,
                       convexhull,
                       hcartesianproduct, vcartesianproduct,
-                      points,
-                      default_library
+                      points
+    import CDDLib # default backend
```

Comment:
I do not like this. We should not load an optional package. I would rather update the docs by highly recommending to also load `CDDLib` (and maybe even printing a warning if `Polyhedra` is loaded first). Below we can use `CDDLib` as default depending on whether that package is loaded.

Reply:
```suggestion
    import .CDDLib # default backend
```

Reply:
Okay, it does not work like that. New suggestion: We do *not* load `CDDLib` by default, but we set our default backend to it. When the `default_polyhedra_backend` function is called, we check if the backend is `CDDLib`, and if so, we check whether `CDDLib` is loaded. If it is not loaded, we print an error message, saying that the user should either load `CDDLib` or should change the backend actively.